pkg/drivers/http/options.go

```diff
@@ -125,3 +129,21 @@ func WithCookies(cookies []drivers.HTTPCookie) Option {
 		}
 	}
 }
+
+func WithAllowedHTTPCode(httpCode int) Option {
```

Comment:
I think WithAllowedHTTPCode is excess here. We already have WithAllowedHTTPCodes

Reply:
I added two functions because all the other Options come in two variants, one singular and one plural. But yes, in my opinion one vararg function would be cleaner: `func WithAllowedHTTPCodes(httpCodes ...int)`. I left it as-is for now.

Reply:
I think 2 functions is ok.